docs/credits.md

```diff
@@ -1223,7 +1223,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | xtend                                             | Raynos                                                                    | [https://github.com/Raynos/xtend](https://github.com/Raynos/xtend)                                                                                                                                                          | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | y18n                                              | Ben Coe                                                                   | [https://github.com/yargs/y18n](https://github.com/yargs/y18n)                                                                                                                                                              | [ISC](https://opensource.org/licenses/ISC)                                                                          |
 | yallist                                           | Isaac Z. Schlueter                                                        | [https://github.com/isaacs/yallist](https://github.com/isaacs/yallist)                                                                                                                                                      | [ISC](https://opensource.org/licenses/ISC)                                                                          |
-| YamlDotNet                                        | Antoine Aubry                                                             | [https://github.com/aaubry/YamlDotNet/wiki](https://github.com/aaubry/YamlDotNet/wiki)                                                                                                                                      | [MIT](https://github.com/aaubry/YamlDotNet/blob/master/LICENSE)                                                     |
+| YamlDotNet                                        | Antoine Aubry                                                             | [https://github.com/aaubry/YamlDotNet/wiki](https://github.com/aaubry/YamlDotNet/wiki)                                                                                                                                      | [Undefined](https://github.com/aaubry/YamlDotNet/blob/master/LICENSE)                                               |
```

Comment:
This doesn't look right - I think the file got renamed to LICENSE.txt and they possibly haven't updated their package manifest accordingly
```suggestion
| YamlDotNet                                        | Antoine Aubry                                                             | [https://github.com/aaubry/YamlDotNet/wiki](https://github.com/aaubry/YamlDotNet/wiki)                                                                                                                                      | [MIT](https://github.com/aaubry/YamlDotNet/blob/master/LICENSE.txt)                                               |
```